src/pool/srv_pool.c

```diff
@@ -2627,6 +2627,7 @@ ds_pool_list_cont_handler(crt_rpc_t *rpc)
 void
 ds_pool_query_handler(crt_rpc_t *rpc)
 {
+	D_PRINT("[RYON] %s:%d [%s()] > \n", __FILE__, __LINE__, __FUNCTION__);
```

Comment:
(style)  unnecessary whitespace before a quoted newline
(style)  __func__ should be used instead of gcc specific __FUNCTION__